sw/tools/sbus_fakerator/sbus_fakerator

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+python sw/tools/sbus_fakerator/main.py $1 $2
```

Comment:
sure it works, just saying there is no need to have an extra file for this....
You could have also just added `#!/usr/bin/env python` to the top of main.py and renamed that to sbus_fakerator...